internal/cmd/schema-registry/command_schema_create.go

```diff
@@ -56,6 +57,9 @@ func (c *command) newSchemaCreateCommand() *cobra.Command {
 	cmd.Flags().String("subject", "", SubjectUsage)
 	pcmd.AddSchemaTypeFlag(cmd)
 	cmd.Flags().String("references", "", "The path to the references file.")
+	cmd.Flags().String("metadata", "", "The path to metadata file.")
+	cmd.Flags().String("rule-set", "", "The path to schema rule set file.")
```

Comment:
"ruleset" is one word
```suggestion
	cmd.Flags().String("ruleset", "", "The path to schema ruleset file.")
```